django/cantusdb_project/main_app/templates/400.html

```diff
@@ -0,0 +1,39 @@
+{% load static %}
+<html lang="en">
```

Comment:
Is this file still used too?

Reply:
Our static file is used to load the background image displayed on the custom error pages.

Reply:
I am also a bit confused by this. It looks like `400.html` is introduced with this PR, but not referenced as far as I can tell anywhere else in the changes? Does django look for a `400.html` by default?

Reply:
I spent a while verifying what is happening with this before removing the error handlers, and found that when Django encounters a 400 error (or 404, etc.) it will locate the 400.html page in the template directory by default. The error handlers are for the case where you might want to customize the error view. Django is able to serve the correct html files even if they are not referenced.

https://docs.djangoproject.com/en/5.0/ref/views/#the-400-bad-request-view